test/Enable.Extensions.Queuing.RabbitMQ.Tests/RabbitMQTestFixture.cs

```diff
@@ -37,11 +37,13 @@ public RabbitMQTestFixture()
 
         public void ClearQueue()
         {
-            using (var connection = _connectionFactory.CreateConnection())
-            using (var channel = connection.CreateModel())
-            {
-                channel.QueuePurge(QueueName);
-            }
+            var connection = _connectionFactory.CreateConnection();
```

Comment:
Why have the `using` statements been removed here? We should retain these so that `Dispose` is always called. With the current change, we're not disposing of the channel, nor would we dispose of the connection if `channel.QueuePurge` were to throw.